Bottom: 966da3cbfb221fd1256d871f19aed5fa9f087efe
Top:    743466bb50222ac9111b6a2bdf7d053cd05b6e2e
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-27 12:06:20 -0400

Set the MPC5200 i2s driver to BROKEN status.

Set the MPC5200 i2s driver to BROKEN status.

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 9fc9082..e7dd79a 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -21,7 +21,7 @@ config SND_SOC_MPC8610_HPCD
 
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
-	depends on PPC_MPC52xx && PPC_BESTCOMM
+	depends on PPC_MPC52xx && PPC_BESTCOMM && BROKEN
 	select SND_SOC_OF_SIMPLE
 	select PPC_BESTCOMM_GEN_BD
 	help
